src/components/ProjectSearch/ProjectSearch.js

```diff
@@ -1,180 +0,0 @@
-// @flow
-
-import { DOM as dom, PropTypes, Component, createFactory } from "react";
-import { connect } from "react-redux";
-import { bindActionCreators } from "redux";
-import actions from "../../actions";
-import { getSources, getActiveSearchState } from "../../selectors";
-import { endTruncateStr } from "../../utils/utils";
-import { parse as parseURL } from "url";
-import { isPretty } from "../../utils/source";
-import { isEnabled } from "devtools-config";
```

Comment:
ProjectSearch now is a coordinator, it just decides which search UI to show and will eventually have buttons to switch between them